lib/pure/math.nim

```diff
@@ -317,6 +326,14 @@ else:
 
 {.pop.}
 
+proc degToRad*[T](d: T): float {.inline.} =
```

Comment:
I don't think these should be generic. Also I agree there should be float32/float64 variants instead.